traffic_ops/traffic_ops_golang/server/put_status.go

```diff
@@ -156,10 +157,12 @@ func updateServerStatusAndOfflineReason(serverID, statusID int, offlineReason *s
 	q := `
 UPDATE server
 SET    status = $1,
-       offline_reason = $2
-WHERE  id = $3
+       offline_reason = $2,
+       status_last_updated = $3
```

Comment:
You can actually just set this to `NOW()` right in the query, no need to interpolate.